lib/pipeline/addon/pipelines/project/pipeline/controller.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  queryParams: ['mode'],
+  mode:'',
+  growl: Ember.inject.service(),
+  stagges: function() {
+    var pipeline = this.get('model.pipeline');
+    return pipeline.stages
+  }.property('model'),
+  errors: null,
+  filteredPipelineHistory: function(){
+    let pipelineHistory = this.get('model.pipelineHistory');
+    let pipeline = this.get('model.pipeline');
+    if(!pipelineHistory){
+      return;
+    }
+    let filteredPipelineHistory = pipelineHistory
+      .filter(ele=>ele.pipeline.id === pipeline.id)
+        .sort((a,b)=>new Date(b.status.started) - new Date(a.status.started));
```

Comment:
`new Date()` is relatively slow.  add a computed property on the model so it can be computed once instead of many times during sort.